sdks/builds/determine-linux-flavor.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+if [ -f /etc/os-release ]; then
```

Comment:
Why isn't `lsb_release -s -i` sufficient?

Reply:
because it may not exist on some systems. LSB is being "phased out"

Reply:
Is XA supporting any distribution which doesn't have `lsb_release`?

Reply:
I honestly don't know... We support "Linux" in general - I'd like to stay as open with the support as possible, but I don't want to make assumptions about something working or not on other distributions than the ones I (or somebody else) tested.